### ISSUE TITLE
Duplicate value in Item 

### ISSUE THREAD

**Issue by 81093507689** [hackiftekhar/IQDropDownTextField#122]
in textfield i have 3 items 
["A","B","A"]
at that time when i select third option its going automatic first option

**Comment by hackiftekhar** [comment]
That's correct. You should set unique items in the list, otherwise you may have to modify it a little bit something like below by adding an space etc
["A", "B", "A "]

and tackle the selection by it's index
